### PR TITLE
Add FPSController Files into Development

### PR DESCRIPTION
**Pull request by iDerp**

### CODE REVIEW

**Comment by 4qua** [comment]
There is no need to move these files back into development. We've already stated over and over again, that the FPSController and the FPS gamemode are complete. The branch 'testing ' is where these files should be, as testing is the evaluation phase, where final tweaks come into play, and these files need simple adjustments that would waste a lot of time if we push these files back into development. The files are complete, and are ready for testing to see what minor adjustments are to be made. This is what the testing branch is for, which is why I placed the files in this branch. End of pull request, alright? I don't mean to be a douche here, I sincerely mean it. It's just your reasoning to push these files all the way back into development is invalid, and it's be a COMPLETE waste of time and effort, for some numerical value shifting here and there. I hope everyone uses this as an example for valid pull requests. Sorry to sound like a complete asswipe, but if this keeps happening, this'll be one hell of a GitMergeClusterfuck.

**Comment by 4qua** [comment]
tl;dr - No reason for a pull request; would be a waste of time in the long run.
